examples/aura.yaml

```diff
@@ -0,0 +1,106 @@
+participants:
+  - el_client_type: nethermind
+    el_client_image: nethermind/nethermind:kurtosis
+    el_client_log_level: ""
+    el_extra_params: [
+      "--config=/nethermind/configs/validator.cfg",
+      "--Init.ChainSpecPath=/nethermind/chainspec/chainspec.json"
+    ]
+    cl_client_type: na
```

Comment:
Is this still needed?

Reply:
It can be removed, but the node name will be `el-1-nethermind-lighthouse` I guess is the default value.